### ISSUE TITLE
Document use of Newick in all opentree tools

### ISSUE THREAD

**Issue by jar398** [OpenTreeOfLife/germinator#13]
See
https://github.com/OpenTreeOfLife/treemachine/issues/147

treemachine, peyotl, and smasher all generate Newick and have to make decisions on how to export OTT taxon 'names' to the subset of Newick (http://evolution.genetics.washington.edu/phylip/newick_doc.html) that dendropy and NEXUS (http://sysbio.oxfordjournals.org/content/46/4/590.full.pdf) handle.

**Comment by jar398** [comment]
See discussion here http://irclog.perlgeek.de/opentreeoflife/2015-01-26

**Comment by jar398** [comment]
I'm using @josephwb's newickName method, which says it matches http://evolution.genetics.washington.edu/phylip/newick_doc.html .  I'm going to assume this is right and close the issue.

**Comment by jar398** [comment]
oops meant to close https://github.com/OpenTreeOfLife/reference-taxonomy/issues/122

**Comment by josephwb** [comment]
I don't think you should use my `newickName` method. otcetera complains a lot about the names in the synthetic tree; @mtholder will have a better method.

**Comment by josephwb** [comment]
This seems relevant: https://github.com/OpenTreeOfLife/opentree/issues/443
